packages/google-pay-integration/src/google-pay-payment-strategy.ts

```diff
@@ -69,11 +69,12 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         await this._paymentIntegrationService.submitOrder();
 
         const nonce = await this._googlePayPaymentProcessor.getNonce(payment.methodId);
+        const extraData = await this._googlePayPaymentProcessor.getExtraData();
```

Comment:
It might be better to update the name to `extraPaymentData` to understand where it should be used